server/app/featureflags/FeatureFlags.java

```diff
@@ -42,80 +33,12 @@ public boolean areOverridesEnabled() {
         && config.getBoolean(FeatureFlag.FEATURE_FLAG_OVERRIDES_ENABLED.toString());
   }
 
-  /**
-   * If the Eligibility Conditions feature is enabled.
-   *
-   * <p>Allows for overrides set in {@code request}.
-   */
-  // TODO(#4437): remove and have clients call getFlagEnabled directly.
-  public boolean isProgramEligibilityConditionsEnabled(Request request) {
-    return getFlagEnabled(request, PROGRAM_ELIGIBILITY_CONDITIONS_ENABLED);
-  }
-
-  /** If the Eligibility Conditions feature is enabled in the system configuration. */
-  // TODO(#4437): remove and have clients call getFlagEnabled directly.
-  public boolean isProgramEligibilityConditionsEnabled() {
-    return config.getBoolean(PROGRAM_ELIGIBILITY_CONDITIONS_ENABLED.toString());
-  }
-
   /** If the reporting view in the admin UI is enabled */
   // TODO(#4437): remove and have clients call getFlagEnabled directly.
```

Comment:
What issue should this be pointing at?

Reply:
There isn't one; I put my username to assign responsibility to remove this in a bit.